.gitignore

```diff
@@ -1,3 +1,5 @@
 /target/
 .idea
 apikeys.js
+node_modules/
+package-lock.json
```

Comment:
Can you add comments to each of these, like this one: https://github.com/googleinterns/step167-2020/blob/master/.gitignore